chromium_src/components/signin/public/identity_manager/identity_manager.cc

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/chromium_src/components/signin/public/identity_manager/identity_manager.h"
```

Comment:
Should just be `components/signin/public/identity_manager/identity_manager.h`?

Reply:
Fixed